src/event/util/log/insert.rs

```diff
@@ -45,33 +51,39 @@ where
             while values.len() <= current {
                 values.push(Value::Null);
             }
-            values[current] = value;
+            let mut swapper = value;
+            std::mem::swap(&mut values[current], &mut swapper);
+            Some(swapper)
```

Comment:
Would this be better as `Some(std::mem::replace(&mut values[current], value))`?

Reply:
I think you're right